SECURITY.md

```diff
@@ -0,0 +1,30 @@
+# Security Policy
+
+## Supported Versions
+
+Use this section to tell people about which versions of your project are
+currently being supported with security updates.
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 2.x   | :x:                |
+| 3.x   | :white_check_mark: |
+| 4.x   | :white_check_mark: |
+
+## Reporting a Vulnerability
+
+At Verdaccio, we consider the security of our systems a top priority. But no matter how much effort we put into system security, there can still be vulnerabilities present. If you've discovered a vulnerability, please follow the guidelines below to report it to our team:
+
+* Report it either [Snyk Security Team](https://snyk.io/vulnerability-disclosure/) or [npmjs Security Team](https://www.npmjs.com/advisories/report?package=verdaccio), they will in contact with us in case of confirm the vulnerability.
```

Comment:
they will in contact => they will be in contact

of confirm => of confirmation of
Or better: in the case that the vulnerability is confirmed.

Reply:
Actually, the point here is, they can contact us directly, but I'm sure their security teams have better tooling and experience testing the patch we can provide.